src/app/common/validation/forms/address-validators.ts

```diff
@@ -21,16 +21,17 @@ export function notCurrentAddressValidator(currentAddress: string) {
 export function btcAddressValidator() {
   return yup
     .string()
-    .defined('Enter a Bitcoin address')
+    .defined(FormErrorMessages.AddressRequired)
```

Comment:
Checking whether we want [defined](https://github.com/jquense/yup#schemadefined-schema) or [required](https://github.com/jquense/yup#schemarequiredmessage-string--function-schema) here?

Reply:
Good question, I just changed the error message itself. Is `required` better in this case over `defined`?